<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMedDialog</class>
 <widget class="QDialog" name="AddMedDialog">
  <property name="geometry">
   <rect>
	<x>0</x>
	<y>0</y>
	<width>481</width>
	<height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
	<widget class="QLabel" name="name_label">
	 <property name="minimumSize">
	  <size>
	   <width>0</width>
	   <height>30</height>
	  </size>
	 </property>
	 <property name="font">
	  <font>
	   <pointsize>16</pointsize>
	  </font>
	 </property>
	 <property name="text">
	  <string>Название</string>
	 </property>
	</widget>
   </item>
   <item row="3" column="0" colspan="2">
	<widget class="QLabel" name="freq_label">
	 <property name="minimumSize">
	  <size>
	   <width>0</width>
	   <height>30</height>
	  </size>
	 </property>
	 <property name="font">
	  <font>
	   <pointsize>16</pointsize>
	  </font>
	 </property>
	 <property name="text">
	  <string>Частота приема</string>
	 </property>
	</widget>
   </item>
   <item row="5" column="0" colspan="2">
	<widget class="QLabel" name="note_label">
	 <property name="minimumSize">
	  <size>
	   <width>0</width>
	   <height>30</height>
	  </size>
	 </property>
	 <property name="font">
	  <font>
	   <pointsize>16</pointsize>
	  </font>
	 </property>
	 <property name="text">
	  <string>Примечание</string>
	 </property>
	</widget>
   </item>
   <item row="6" column="0">
	<spacer name="horizontalSpacer">
	 <property name="orientation">
	  <enum>Qt::Horizontal</enum>
	 </property>
	 <property name="sizeHint" stdset="0">
	  <size>
	   <width>40</width>
	   <height>20</height>
	  </size>
	 </property>
	</spacer>
   </item>
   <item row="0" column="2">
	<widget class="QLineEdit" name="name_lineEdit">
	 <property name="minimumSize">
	  <size>
	   <width>0</width>
	   <height>30</height>
	  </size>
	 </property>
	 <property name="font">
	  <font>
	   <pointsize>16</pointsize>
	  </font>
	 </property>
	</widget>
   </item>
   <item row="3" column="2">
	<widget class="QLineEdit" name="freq_lineEdit">
	 <property name="minimumSize">
	  <size>
	   <width>0</width>
	   <height>30</height>
	  </size>
	 </property>
	 <property name="font">
	  <font>
	   <pointsize>16</pointsize>
	  </font>
	 </property>
	</widget>
   </item>
   <item row="2" column="2">
	<widget class="QComboBox" name="type_comboBox">
	 <property name="minimumSize">
	  <size>
	   <width>0</width>
	   <height>30</height>
	  </size>
	 </property>
	 <property name="font">
	  <font>
	   <pointsize>16</pointsize>
	  </font>
	 </property>
	</widget>
   </item>
   <item row="6" column="1" colspan="2">
	<widget class="QDialogButtonBox" name="buttonBox">
	 <property name="orientation">
	  <enum>Qt::Horizontal</enum>
	 </property>
	 <property name="standardButtons">
	  <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
	 </property>
	</widget>
   </item>
   <item row="1" column="2">
	<widget class="QLineEdit" name="dose_lineEdit">
	 <property name="minimumSize">
	  <size>
	   <width>0</width>
	   <height>30</height>
	  </size>
	 </property>
	 <property name="font">
	  <font>
	   <pointsize>16</pointsize>
	  </font>
	 </property>
	</widget>
   </item>
   <item row="5" column="2">
	<widget class="QLineEdit" name="note_lineEdit">
	 <property name="minimumSize">
	  <size>
	   <width>0</width>
	   <height>30</height>
	  </size>
	 </property>
	 <property name="font">
	  <font>
	   <pointsize>16</pointsize>
	  </font>
	 </property>
	</widget>
   </item>
   <item row="2" column="0" colspan="2">
	<widget class="QLabel" name="type_label">
	 <property name="minimumSize">
	  <size>
	   <width>0</width>
	   <height>30</height>
	  </size>
	 </property>
	 <property name="font">
	  <font>
	   <pointsize>16</pointsize>
	  </font>
	 </property>
	 <property name="text">
	  <string>Тип</string>
	 </property>
	</widget>
   </item>
   <item row="1" column="0" colspan="2">
	<widget class="QLabel" name="dose_label">
	 <property name="minimumSize">
	  <size>
	   <width>0</width>
	   <height>30</height>
	  </size>
	 </property>
	 <property name="font">
	  <font>
	   <pointsize>16</pointsize>
	  </font>
	 </property>
	 <property name="text">
	  <string>Дозировка</string>
	 </property>
	</widget>
   </item>
   <item row="4" column="2">
	<widget class="QLineEdit" name="term_lineEdit">
	 <property name="minimumSize">
	  <size>
	   <width>0</width>
	   <height>30</height>
	  </size>
	 </property>
	 <property name="font">
	  <font>
	   <pointsize>16</pointsize>
	  </font>
	 </property>
	</widget>
   </item>
   <item row="4" column="0" colspan="2">
	<widget class="QLabel" name="term_label">
	 <property name="minimumSize">
	  <size>
	   <width>0</width>
	   <height>30</height>
	  </size>
	 </property>
	 <property name="font">
	  <font>
	   <pointsize>16</pointsize>
	  </font>
	 </property>
	 <property name="text">
	  <string>Длительность приема</string>
	 </property>
	</widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name_lineEdit</tabstop>
  <tabstop>dose_lineEdit</tabstop>
  <tabstop>type_comboBox</tabstop>
  <tabstop>freq_lineEdit</tabstop>
  <tabstop>term_lineEdit</tabstop>
  <tabstop>note_lineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddMedDialog</receiver>
   <slot>accept()</slot>
   <hints>
	<hint type="sourcelabel">
	 <x>248</x>
	 <y>254</y>
	</hint>
	<hint type="destinationlabel">
	 <x>157</x>
	 <y>274</y>
	</hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddMedDialog</receiver>
   <slot>reject()</slot>
   <hints>
	<hint type="sourcelabel">
	 <x>316</x>
	 <y>260</y>
	</hint>
	<hint type="destinationlabel">
	 <x>286</x>
	 <y>274</y>
	</hint>
   </hints>
  </connection>
 </connections>
</ui>
